### DISCUSSION TITLE
How to scale dokku infrastructure vertically?

### DISCUSSION THREAD

**Discussion by dimasna** [dokku/dokku#5203] [Q&A]
What cloud provider that enable dokku to scale vertically? so imagine that I have 1 GB ram for dokku and I have 2 app, and I allocate 512mb ram for each of my app. and when I want to deploy new app and I also allocate 512mb ram for it, I want the cloud provider auto scale my original ram (1GB) to 1.5GB or 2GB? is it possible?

**Comment by gschurck** [comment]
You will need to automate this process manually with a cloud provider API (but of course your server will be restarted to scale up)
